src/content/docs/guides/configure-biome.mdx

```diff
@@ -0,0 +1,391 @@
+:::
+
+
+## Well-known files
+
+Here are some well-known files that we specifically treat based on their file names, rather than their extensions.
+Currently, the well-known files are JSON-like files only, but we may broaden the list to include other types when we support new parsers.
+
+The following files are parsed as `JSON` files with both the options `json.parser.allowComments` and `json.parser.allowTrailingCommas` set to `false`.
+
+- `.all-contributorsrc`
+- `.arcconfig`
+- `.auto-changelog`
+- `.bowerrc`
+- `.c8rc`
+- `.htmlhintrc`
+- `.imgbotconfig`
+- `.jslintrc`
+- `.nycrc`
+- `.tern-config`
+- `.tern-project`
+- `.vuerc`
+- `.watchmanconfig`
+- `mcmod.info`
+
+The following files are parsed as `JSON` files with the options `json.parser.allowComments` set to `true` but `json.parser.allowTrailingCommas` set to `false`. This is because the tools consuming these files can only strip comments.
+
+- `.ember-cli`
+- `.eslintrc.json`
+- `.jscsrc`
+- `.jshintrc`
+- `tslint.json`
+
+The following files are parsed as `JSON` files with the options `json.parser.allowComments` and `json.parser.allowTrailingCommas` set to `true`. This is because the tools consuming these files are designed to accommodate such settings.
+
+- `.babelrc`
+- `.babelrc.json`
+- `.devcontainer.json`
+- `.hintrc`
+- `.hintrc.json`
+- `.swcrc`
+- `api-documenter.json`
+- `api-extractor.json`
+- `babel.config.json`
+- `deno.json`
+- `devcontainer.json`
+- `dprint.json`
+- `jsconfig.json`
+- `jsr.json`
+- `language-configuration.json`
+- `tsconfig.json`
+- `typedoc.json`
+- `typescript.json`
+
+
+## Glob syntax explained
+
+A glob pattern specifies a set of filenames.
+Biome supports the following globs:
+
+- `*` matches zero or more characters. It cannot match the path separator `/`.
+- `**` recursively matches directories and files.
+  This sequence **must** form a single path component,
+  so both `**a` and `b**` are invalid and will result in an error.
+  A sequence of more than two consecutive `*` characters is also invalid.
+- `[...]` matches any character inside the brackets.
+  Ranges of characters can also be specified, as ordered by Unicode, so e.g.
+  `[0-9]` specifies any character between 0 and 9 inclusive.
+- `[!...]` is the negation of `[...]`, i.e. it matches any characters **not** in the brackets.
+
+Some examples:
+
+- `dist/**` matches the dist directory and all files in this directory.
+- `**/test/**` matches all files under any directory named `test`, regardless of where they are. E.g. `dist/test`, `src/test`.
+- `**/*.js` matches all files ending with the extension `.js` in all directories.
+
+Biome uses a glob library that treats all globs as having a `**/` prefix.
+This means that `src/**/*.js` and `**/src/**/*.js` are treated as identical.
+They match both `src/file.js` and `test/src/file.js`.
+This is something we plan to fix in Biome **v2.0.0**, as it will be a breaking change.
```

Comment:
There's no need to mention "future" plans inside docs:
- it forces us to remember to remove this line when there's a new release
- it "expires" and confuses people
- feels like a `TODO`

When the logic changes, we change the explanation accordingly.